### ISSUE TITLE
[Bug]: Tags and links hard to read in darkmode

### ISSUE THREAD

**Issue by sytone** [RafaelGB/obsidian-db-folder#809]
### Contact Details

github@sytone.com

### What happened?

When using a db view and using lookup or tags the font colour is hard to read against the background. I have only come back to folder db recently and do not remember this from before.

Tries Default and Minimal themes and have the same display issue.

I did find similar issues but they were closed as fixed and I confirmed I am on the recent version. So either this is separate or there is something wrong locally and I have disabled everything to go back to default with the same display issue showing.

![image](https://user-images.githubusercontent.com/1399443/217393999-0ff033d1-0869-4011-b9a9-d705d2d478a4.png)

![image](https://user-images.githubusercontent.com/1399443/217394026-d851767e-45f8-4ac0-8c35-c2c386415084.png)

Example config:
      - { label: "Completed", backgroundColor: "hsl(107,79%,52%)"}
      - { label: "Todo", backgroundColor: "hsl(79,73%,49%)"}
      - { label: "In Progress", backgroundColor: "hsl(206,66%,56%)"}
      - { label: "Discarded", backgroundColor: "hsl(300,2%,76%)"}
      - { label: "Waiting", backgroundColor: "hsl(302,65%,53%)"}
      - { label: "Archive", backgroundColor: "hsl(253,40%,68%)"}

### What platform were you using?

Desktop

### Version of the plugin

3.3.0

### Are you using the latest version of the plugin in your Obsidian vault?

- [X] I have verified that I am on the latest version

### Are you check if there is a similar issue?

- [X] I have verified that there are not similar issues

**Comment by RafaelGB** [comment]
Hello @sytone ,
The color pf the text depends directly pf the used theme. To support all kind of configurations I was thinking about to add a new color property to the tags column to select the text color

**Comment by GaboCapo** [comment]
Here is a little Workaround with CSS Fix for that Issue, tested for the "Minimal" Theme.
just add a file e.g. `cssfix.css` in `snippets` Folder in your .obsidian Folder. 
Reference:
https://help.obsidian.md/Extending+Obsidian/CSS+snippets

```
.markdown-preview-view.database-plugin__markdown-preview-view {
    color: #606060;
}
```

**Comment by RafaelGB** [comment]
Commit db3839f7 fixed =)

**Comment by cesarpereira904** [comment]
Hi @RafaelGB @GaboCapo 

Another suggestion, because Relationlinks can have a background just like tags, I find the default hyperlinks do not fit most of the background colors I set:

![Image](https://user-images.githubusercontent.com/70922426/219614738-1bd33efe-befd-44b0-90ac-90e8f8b86853.png)


I would suggest for links to have same default text color:

![Image](https://user-images.githubusercontent.com/70922426/219614807-f4d4e7ba-ddfb-4eb7-b2e8-9d04061b8e92.png)

.markdown-preview-view.database-plugin__markdown-preview-view a:link{
    color: #dadada !important;
    text-decoration: none;
}

and to be able to distinguish a link from normal text to introduce a new symbol, just like this example:


![Image](https://user-images.githubusercontent.com/70922426/219614945-2bf9dfd6-4fe9-4059-8419-ea1c14ed34b6.png)



what do you think?

**Comment by RafaelGB** [comment]
Could be a good option, sure!

**Comment by RafaelGB** [comment]
Included with next version

```css
.database-plugin__relation-container .database-plugin__relationship a:link {
  color: #dadada;
  text-decoration: none;
}

.database-plugin__relation-container .database-plugin__relationship a:hover {
  color: #a38484;
}
```

**Comment by cyb3rw0lf** [comment]
how to separate the color for normal text fields and fields with the background?
In dark theme if I choose a dark color it will be applied on both so either the text field or the ones with light background will be barely readable.

```css
.markdown-preview-view.database-plugin__markdown-preview-view {
  color: #494949;
}
```
<img width="604" alt="darkthemeissue" src="https://user-images.githubusercontent.com/25265715/220873419-2aadbdfe-d1c9-46f7-a844-8c18acc955b8.png">


```css
.markdown-preview-view.database-plugin__markdown-preview-view {
  color: #dadada;
}
```
<img width="596" alt="darkthemeissuelight" src="https://user-images.githubusercontent.com/25265715/220873708-88b843f1-078b-4dc4-b533-e000706e151f.png">

**Comment by RafaelGB** [comment]
available with 3.3.2
